calibrate-touchscreen/src/main.rs

```diff
@@ -76,6 +76,7 @@ impl FullscreenSurface {
         );
 
         self.wl_surface.attach(Some(&buffer), 0, 0);
+        self.wl_surface.damage(0, 0, width, height);
```

Comment:
This should not have the effect you observe. Setting `damage` is optional, and only useful when the damage area is smaller than the buffer (it allows the compositor to reduce the area repainted).